tests/test_exceptions.py

```diff
@@ -0,0 +1,304 @@
+from typing import Dict, List, Union
+from uuid import UUID
+
+import pytest
+
+from pydantic import BaseModel
+from pydantic.exceptions import ValidationError, flatten_errors
+
+
+class SubModel(BaseModel):
+    x: int
+    y: int
+    z: str
+
+
+class Model(BaseModel):
+    a: int
+    b: SubModel
+    c: List[SubModel]
+    d: Union[int, UUID]
+    e: Dict[int, str]
+    f: List[Union[int, str]]
+
+
+def test_validation_error_display_errors():
+    with pytest.raises(ValidationError) as exc_info:
```

Comment:
I think maybe of these tests can be rewritten as one parameterized test.

Reply:
fixed